runtime/doc/develop.txt

```diff
@@ -667,5 +667,22 @@ External UIs are expected to implement these common features:
 - Support the text decorations/attributes given by |ui-event-hl_attr_define|.
   The "url" attr should be presented as a clickable hyperlink.
 
+Multigrid UI ~
+                                                        *dev-ui-multigrid*
+- A multigrid UI should display floating windows using one of the following
```

Comment:
could reference this section from `win_float_pos` or just the intro of `ui-multigrid` in ui.txt

Reply:
I added the reference